spec/features/workflow/unpublishing_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "rake"
+
+Rails.application.load_tasks
```

Comment:
I think this will have an effect for all the specs - would be good to move this somewhere central so it's clear it applies globally.

Reply:
I'll move it to `spec_helper`

Reply:
Fixed up.